src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,49 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    public static final int FIGURE_COUNT = 5;
+    private ColorSupplier cs = new ColorSupplier();
+    private Random random = new Random();
+
+    public Figure getRandomFigure() {
+        int shapeNumber = random.nextInt(FIGURE_COUNT);
+        double firstFigureSide = random.nextInt(20);
+        double secondFigureSide = random.nextInt(20);
+        double thirdFigureSide = random.nextInt(20);
+        Figure figure = null;
```

Comment:
this is a redundant variable, you can return Figure in case-block directly. PS. After that you can remove break statements 
```suggestion
        Figure figure = null;
```